zebra-instructions.md

```diff
@@ -75,11 +70,11 @@ These fixes disable mining pool operator payments and miner payments: they just
     miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
     ```
     </details>
-3. [Run Zebra](https://zebra.zfnd.org/user/run.html) with the `getblocktemplate-rpcs` feature:
+2. [Build](https://github.com/ZcashFoundation/zebra#build-instructions) and [Run Zebra](https://zebra.zfnd.org/user/run.html) with the `getblocktemplate-rpcs` feature:
     ```sh
-    cargo run --release --features "getblocktemplate-rpcs"
+    cargo run --release --features "getblocktemplate-rpcs" --bin zebrad -- -c zebrad.toml
```

Comment:
Should we leave a `zebrad.toml` config in the root folder of this repo instead of adding it as an example in the instructions?